fireatlas/postprocess.py

```diff
@@ -347,11 +383,21 @@ def save_large_fires_layers(allfires_gdf, region, large_fires, tst, ted):
     # we'll set the "fireID" to "mergeid" in those spots
     gdf.loc[merge_needed, "fireID"] = gdf.loc[merge_needed, "mergeid"]
 
+    processed_gdfs = []
     for fid, data in gdf[gdf["fireID"].isin(large_fires)].groupby("fireID"):
         # merge any rows that have the same t
         if data.t.duplicated().any():
             data = merge_rows(data)
-        save_fire_layers(data, region, int(fid), tst)
+
+        if combine_all:
+            processed_gdfs.append(data)
+        else:
+            save_fire_layers(data, region, int(fid), tst)
```

Comment:
Seems like `save_fire_layers` could also use `create_snapshot_data`

Reply:
hmm, maybe I missed this

Reply:
given that `save_fire_layers` is targeting `columns = [col for col in singlefire_getdd(layer)]` a different schema and only handful of the login in `create_snapshot_data` I think we don't have to be DRY here. Let me know if you disagree strongly

Reply:
ooh I always forget those two are different you are totally right.